raft/src/main/java/org/apache/kafka/raft/KafkaRaftClient.java

```diff
@@ -3315,10 +3315,7 @@ private long pollFollowerAsVoter(FollowerState state, long currentTimeMs) {
 
         return Math.min(
             backoffMs,
-            Math.min(
-                state.remainingFetchTimeMs(currentTimeMs),
-                state.remainingUpdateVoterSetPeriodMs(currentTimeMs)
-            )
+            state.remainingFetchTimeMs(currentTimeMs)
```

Comment:
I see, the backoffMs accounts for the time to wait before processing the result of any updateVoteRequest